### DISCUSSION TITLE
HandShake error

### DISCUSSION THREAD

**Discussion by gastonrey** [visciang/telegram#109] [Q&A]
I'm trying to create a Bot and as soon as it tries to make getMe request I receive next error on console until timed out:

`[info] TLS :client: In state :hello received SERVER ALERT: Fatal - Handshake Failure`

I suspect it's related with new Tesla upgrade missing?

Versions ma using:

Elixir: 1.11
ERL: 23
Telegram: 0.7.1
Cowlib: 2.9

**Comment by visciang** [comment]
HI @gastonrey !
Just tested on 0.7.1. It works on my machine ;) !!

About the update, give it a try, why not.
But it doesn't sound to me it could be related.

... maybe going down into the SSL client/server certificate.

If you are running on your host, try to switch to an elixir docker image and test from there.
